examples/asr/conf/config.yaml

```diff
@@ -1,11 +1,40 @@
 #model: "QuartzNet15x5"
 sample_rate: &sample_rate 16000
-accumulate_grad_batches: &accumulate_grad_batches 1
+repeat: &repeat 1
+dropout: &dropout 0.0
+separable: &separable true
 
-optim_params:
-  name: adam 
+optim:
+  name: novograd
```

Comment:
can this name be replaced with cls like NovogradParams below?

Reply:
name is to use NeMo registered optimizer/scheduler while cls is used to instantiate. So you could have:
```
optim:
  cls: nemo.core.optim.novograd.Novograd
...
```
Correct?

Reply:
I don't want to allow cls in optimizer - because I want to offer dynamic registration of optimizers. Cls doesn't work with dynamic classes as that class would not be in the Nemo class path. So name is the only valid argument for optim instantiation.

Reply:
I think we should add this later as a feature because I think to do this right we'll need better support from Hydra.

Reply:
Tested enough to be happy enough for now. We can replace optim.name with optim.cls safely now.